phoenix-core/src/main/java/org/apache/phoenix/iterate/ScanningResultIterator.java

```diff
@@ -97,9 +95,7 @@ private void changeMetric(GlobalClientMetrics metric, Long value) {
     private void updateMetrics() {
 
         if (scanMetricsEnabled && !scanMetricsUpdated) {
-            ScanMetrics scanMetrics = scan.getScanMetrics();
-            if (scanMetrics == null)
```

Comment:
scanMetrics were expected null in HBase 1.3 only?

Reply:
Yes, 1.3 doesn't have the metrics API that Phoenix uses, it was specifically added for 1.3

Reply:
I realize that we also lose the "scan" class variable, but it is not used, and is overridden in child classes anyway, so that's OK.